py-rse/project.Rmd

```diff
@@ -409,8 +360,8 @@ This lesson will explore who we should write documentation for,
 what we should write for them,
```

Comment:
This sentence begins "this lesson will explore" even though we are half way through the lesson. I suspect it's a hangover from a previous draft and needs to be re-worded.